content/en/samples/posts/2024-12-26-idle-view-app.md

```diff
@@ -0,0 +1,418 @@
+| ![Node.js](https://img.shields.io/badge/-Node.js-339933?logo=node.js&logoColor=white) | Runtime environment for backend development.                                              |
+| ![TypeScript](https://img.shields.io/badge/-TypeScript-3178C6?logo=typescript&logoColor=white) | Enhances JavaScript for type-safe backend development.                                    |
+| ![Express](https://img.shields.io/badge/-Express-000000?logo=express&logoColor=white) | Fast and lightweight web application framework for Node.js.                              |
+
+### Backend Libraries
+| Library                  | Description                                                                                           |
+|--------------------------|-------------------------------------------------------------------------------------------------------|
+| ![Prisma](https://img.shields.io/badge/-Prisma-2D3748?logo=prisma&logoColor=white) | Modern ORM for Node.js and TypeScript to manage database access.                         |
+
+### Database
+| Technology               | Description                                                                                           |
+|--------------------------|-------------------------------------------------------------------------------------------------------|
+| ![MySQL](https://img.shields.io/badge/-MySQL-4479A1?logo=mysql&logoColor=white)     | Relational database management system for structured data storage.                      |
+
+
+---
+
+## Hardware Requirements
+
+To set up this project, you need a **target device** and **host PC**.
+
+### Target Device
+
+You need a Raspberry Pi 4 with webOS OSE as the target device.
+
+| Hardware                        | Description                                                                                              |
+|---------------------------------|----------------------------------------------------------------------------------------------------------|
+| **Raspberry Pi 4 Model B (8GB)** | The core computing unit for the application.                                                             |
+| **MicroSD Card with webOS OSE Image** | A MicroSD for flashing webOS OSE. <br> - webOS OSE 2.24.0 is used in this project. You can get the pre-built image in [webOS OSE GitHub](https://github.com/webosose/build-webos/releases/tag/v2.24.0). <br> - To install a webOS OSE image on a MicroSD card, use the following guide: [Flashing webOS OSE](https://www.webosose.org/docs/guides/setup/flashing-webos-ose/). |
+| **Touchscreen or Monitor**      | The display device that interacts with the application. We recommend using a touchscreen for a more interactive experience. |
+
+### Host PC
+
+Our team developed this project using a Windows environment. However, the project can be set up and executed on other operating systems such as Linux or macOS. The following are the general specifications and software requirements for the host PC:
```

Comment:
Since the audience is reading (in the second person), explanations starting with a subject such as "our team" are unnecessary. You would better suggest a method to set it according to the environment.